Library/Homebrew/tap.rb

```diff
@@ -133,6 +135,33 @@ def install(options = {})
     end
   end
 
+  def link_manpages
+    return unless (path/"man").exist?
+    conflicts = []
+    (path/"man").find do |src|
+      next if src.directory?
+      dst = HOMEBREW_PREFIX/"share"/src.relative_path_from(path)
+      FileUtils.mkdir_p dst.dirname
+      if dst.symlink? && dst.resolved_path.to_s.start_with?("#{path}/man")
+        dst.delete
```

Comment:
I think we should skip in this case instead of deleting then relinking. Since we run it in `brew update`, avoiding unnecessary file operations would be preferred.